content/en/APIUseCases/create_asset.md

```diff
@@ -0,0 +1,229 @@
+---
+title: "Use Our API to Create an Asset"
+linkTitle: "Create an Asset"
+weight: 110
+description: >
+  Use this sequence of REST calls to create an asset.
```

Comment:
Send this sequence...

_Use_ is repeated 3 times.

Reply:
I'm using "Run this sequence..."